packages/@lwc/ssr-compiler/src/compile-js/lwc-import.ts

```diff
@@ -35,3 +35,40 @@ export function replaceLwcImport(path: NodePath<ImportDeclaration>, state: Compo
 
     path.replaceWith(b.importDeclaration(path.node.specifiers, b.literal('@lwc/ssr-runtime')));
 }
+
+/**
+ * This handles lwc barrel exports by replacing "lwc" with "@lwc/ssr-runtime"
+ */
+export function replaceNamedLwcExport(path: NodePath<ExportNamedDeclaration>) {
+    if (!path.node || !isLwcSource(path)) {
+        return;
+    }
+
+    path.replaceWith(
+        b.exportNamedDeclaration(
+            path.node.declaration,
+            path.node.specifiers,
+            b.literal('@lwc/ssr-runtime')
+        )
+    );
+}
+
+/**
+ * This handles all lwc barrel exports by replacing "lwc" with "@lwc/ssr-runtime"
+ */
+export function replaceAllLwcExport(path: NodePath<ExportAllDeclaration>) {
+    if (!path.node || !isLwcSource(path)) {
+        return;
+    }
+
+    path.replaceWith(b.exportAllDeclaration(b.literal('@lwc/ssr-runtime')));
+}
```

Comment:
This doesn't correctly handle the case of 

```js
export * as foo from 'lwc'
```

which is not the same as 

```js
export * from 'lwc'
```

In this case, we would need to maintain the `foo` identifier as well (try this example on https://astexplorer.net/).

Reply:
thank you, sorry I missed this